examples/ui-demo/src/components/shared/7702/MintCard7702.tsx

```diff
@@ -0,0 +1,94 @@
+import Image from "next/image";
+import { LoadingIcon } from "@/components/icons/loading";
+import { Button } from "./Button";
+import { MintStages } from "./MintStages";
+
+type NFTLoadingState = "initial" | "loading" | "success";
+export type MintStatus = {
+  signing: NFTLoadingState;
+  gas: NFTLoadingState;
+  batch: NFTLoadingState;
+};
+
+export const MintCard7702 = ({
+  isLoading,
+  isDisabled,
+  status,
+  nftTransfered,
+  handleCollectNFT,
+  uri,
+}: {
+  isLoading: boolean;
+  isDisabled?: boolean;
+  status: MintStatus;
+  nftTransfered: boolean;
+  handleCollectNFT: () => void;
+  uri?: string;
+}) => {
+  return (
+    <div className="bg-bg-surface-default rounded-lg p-6 w-[326px] h-[478px] flex flex-col shadow-none lg:shadow-smallCard ">
+      {uri ? (
+        <div className="relative flex flex-col items-center">
+          <Image
+            className="mb-4 rounded-xl object-cover h-[222px] w-[326px]"
+            width={326}
+            height={222}
```

Comment:
❓ nit: any reason for setting height & width via both tailwind as well as the Image props?